modules/luci-base/root/etc/init.d/ucitrack

```diff
@@ -35,7 +35,7 @@ register_trigger() {
 			;;
 			*)
 				logger -t "ucitrack" "Setting up non-init /etc/config/$config reload handler: $exec"
-				procd_add_config_trigger "config.change" "$config" "$exec"
+				procd_add_config_trigger "config.change" "$config" $exec
```

Comment:
@jjm2473 - Hm, if we unquote the exec value it will get evaluated by the shell which might be problematic if special chars are contained.

Can you try if the following works as well?

`procd_add_config_trigger "config.change" "$config" /bin/sh -c "$exec"`

Reply:
I think they're equivalent